### DISCUSSION TITLE
Bootstrap Icons <i> tag versus <span>

### DISCUSSION THREAD

**Discussion by smileBeda** [twbs/bootstrap#36067] [General] [answered]
It is not semantically suggested to use `<i>` tag for an icon, for the following reason:

>The `<strong>`/`<b>` and `<em>`/`<i>` tags have exactly the same effect in most web browsers, but there is a fundamental difference between them: `<strong>` and `<em>` have a semantic meaning whereas `<b>` and `<i>` only convey styling information like CSS.
>
>While `<b>` can have simply no effect on a some devices with limited display or when a screen reader software is used by a blind person, `<strong>` will:
>
>Display the text bold in normal browsers
>Speak with lower tone when using a screen reader such as Jaws
>Consequently:
>
>in order to convey semantics, the `<b>` and `<i>` tags shall never be used,
>in order to convey styling information, the `<b>` and `<i>` should be avoided and CSS should be used instead. 

Is there any reasoning behind providing Bootstrap Icons on https://icons.getbootstrap.com/icons/ with an `<i>` tag instead of for example a `<span>`?

Yes, it is perfectly possible to do that when using the Icons in HTML, but it would be easier to just copy it with the proper HTML entity instead of having to change it each time, and I also think it would make the web a better place if the "big" ones (Bootstrap in this case) would just show the examples with the right tags by default.

Please apologise if I somehow am wrong with above thinking approach.

**Comment by WinterSilence** [comment]
reason? it's uses historically, because `<i>`con + `i` shortest than `span`
"right tag" is [custom](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements)

**Comment by mdo** [comment, marked as answer]
Since `<i>` has no semantics, I don't see it being an issue at all for how we're using it. There's no specific value or use case for it anymore, so it's easy to use as folks can re-assign it in their use case for icon as mentioned in another comment. The element ultimately doesn't matter though—use `<span>`, `<div>`, `<i>`, or even a custom HTML element like `<icon>`. Totally up to you :).
